flow/scene_update_context.h

```diff
@@ -152,6 +152,17 @@ class SceneUpdateContext {
   }
   const fuchsia::ui::gfx::MetricsPtr& metrics() const { return metrics_; }
 
+  void set_frame_dimensions(const SkISize& frame_size,
```

Comment:
Maybe add a comment to specify whether `frame_size, frame_depth` are in the logical space or device/physical space.

Reply:
Done